tsdb/tsi1/gen_test.go

```diff
@@ -0,0 +1,10 @@
+//go:generate sh -c "curl -L https://github.com/influxdata/testdata/raw/master/tsi1testdata.tar.gz | tar xz"
```

Comment:
It would probably be better to tag the testdata repo so that we don't have to coordinate changes to master over there with changes here. I would suggest a date-based tag, something like `2020.07.16.0` since there isn't necessarily any special meaning to any particular version.